master/internal/experiment.go

```diff
@@ -581,7 +581,7 @@ func (e *experiment) toV1Job() *jobv1.Job {
 		Name:           e.Config.Name().String(),
 	}
 
-	j.IsPreemptible = config.ReadPreemptionStatus(j.ResourcePool, &e.Config)
+	j.IsPreemptible = config.ReadRMPreemptionStatus(j.ResourcePool) // && true
```

Comment:
what is this comment?

Reply:
I was trying to signify how this is calculated. as of now experiments on their own are always marked as preemptible, hence the constant true, and commands always not preemptible and consequently [the constant false here](https://github.com/determined-ai/determined/pull/3330#discussion_r771533125) 
I thought about linking these const values in some way to account for future changes eg using a shared var or a method on experiments and commands eg `PreemptionEnabled` wdy think?